.github/workflows/ai-code-review.yml

```
@@ -0,0 +1,387 @@
+
+              This may take a few moments. The review will be posted here when complete.
+
+              _Powered by AI Code Review Pipeline_ 🚀`
+            });
+
+      # ========================================================================
+      # OPTION 1: Claude Code CLI (if available and configured)
+      # ========================================================================
+      - name: 🎭 AI Review with Claude (if configured)
+        id: claude-review
+        if: ${{ secrets.ANTHROPIC_API_KEY != '' }}
```

actionlint (1.7.8)

context "secrets" is not allowed here. available contexts are "env", "github", "inputs", "job", "matrix", "needs", "runner", "steps", "strategy", "vars". see https://docs.github.com/en/actions/learn-github-actions/contexts#context-availability for more details

(expression)